android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/configuration/feature/operation/SingleTopReactivateBackStackOperation.kt

```diff
@@ -0,0 +1,16 @@
+package com.badoo.ribs.core.routing.configuration.feature.operation
+
+import android.os.Parcelable
+import com.badoo.ribs.core.routing.configuration.feature.BackStackElement
+
+internal data class SingleTopReactivateBackStackOperation<C : Parcelable>(
```

Comment:
I'd move this and the other internal one back inside the main one as private probably.